client/src/components/ImageCarousel/styles.js

```diff
@@ -1,18 +1,33 @@
-import styled from 'styled-components';
+import styled, { keyframes, css } from 'styled-components';
```

Comment:
I've never seen keyframes used before or CSS imported like this from styled-components as well, you'll have to tell me how those work and what they're doing!